src/components/Requests/Requests.js

```diff
@@ -4,26 +4,38 @@ import UsersContext from '../../Context/UsersContext'
 import './Requests.scss'
 import Card from '../Card/Card'
 
+
+
 export default function Requests() {
-  const { currentUser, users } = useContext(UsersContext)
-  // const requestRoomie = useContext(RequestRoomieContext)
+  const { currentUser, users } = useContext(UsersContext);
+
+  let connections = []
+  
+  const helper = (array, id) => {
```

Comment:
We should change this naming convention.  Making an issue for it now so we can do it during refactor.